x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/process/autodetect/state/TimingStats.java

```diff
@@ -172,9 +193,20 @@ public void writeTo(StreamOutput out) throws IOException {
 
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        return toXContent(builder, false);
+    }
+
+    public XContentBuilder toXContentWithCalculatedFields(XContentBuilder builder) throws IOException {
+        return toXContent(builder, true);
+    }
+
+    private XContentBuilder toXContent(XContentBuilder builder, boolean includeCalculatedFields) throws IOException {
         builder.startObject();
         builder.field(Job.ID.getPreferredName(), jobId);
         builder.field(BUCKET_COUNT.getPreferredName(), bucketCount);
+        if (includeCalculatedFields) {
```

Comment:
see previous comments around taking advantage of `XContent.Params` for this type of situation.

Reply:
Done.